package-lock.json

```diff
@@ -36094,57 +36099,39 @@
       }
     },
     "react-native-onyx": {
-      "version": "git+https://github.com/Expensify/react-native-onyx.git#d73900b7cb7bf82bed77cb6b6baabf8fe2eb3a0e",
-      "from": "git+https://github.com/Expensify/react-native-onyx.git#d73900b7cb7bf82bed77cb6b6baabf8fe2eb3a0e",
+      "version": "git+https://github.com/Expensify/react-native-onyx.git#d7553b95e982ab78f6bb2064f6b0549f0ace94c2",
+      "from": "git+https://github.com/Expensify/react-native-onyx.git#d7553b95e982ab78f6bb2064f6b0549f0ace94c2",
       "requires": {
         "ascii-table": "0.0.9",
-        "expensify-common": "git+https://github.com/Expensify/expensify-common.git#4016a786d66ff38ed76a191edef31ebd737a0190",
+        "expensify-common": "git+https://github.com/Expensify/expensify-common.git#2e5cff552cf132da90a3fb9756e6b4fb6ae7b40c",
         "lodash": "4.17.21",
         "underscore": "^1.13.1"
       },
       "dependencies": {
-        "classnames": {
-          "version": "2.3.1",
-          "resolved": "https://registry.npmjs.org/classnames/-/classnames-2.3.1.tgz",
-          "integrity": "sha512-OlQdbZ7gLfGarSqxesMesDa5uz7KFbID8Kpq/SxIoNGDqY8lSYs0D+hhtBXhcdB3rcbXArFr7vlHheLk1voeNA=="
-        },
         "expensify-common": {
-          "version": "git+https://github.com/Expensify/expensify-common.git#4016a786d66ff38ed76a191edef31ebd737a0190",
-          "from": "git+https://github.com/Expensify/expensify-common.git#4016a786d66ff38ed76a191edef31ebd737a0190",
+          "version": "git+https://github.com/Expensify/expensify-common.git#2e5cff552cf132da90a3fb9756e6b4fb6ae7b40c",
+          "from": "git+https://github.com/Expensify/expensify-common.git#2e5cff552cf132da90a3fb9756e6b4fb6ae7b40c",
           "requires": {
-            "classnames": "2.3.1",
+            "classnames": "2.2.5",
             "clipboard": "2.0.4",
             "html-entities": "^1.3.1",
-            "jquery": "3.6.0",
+            "jquery": "3.3.1",
```

Comment:
Is there a reason we're downgrading to an older release of jQuery?

Reply:
It happened automatically and it seem to be the correct version acording to package specification
Onyx uses Expensify/expensify-common@2e5cff552cf132da90a3fb9756e6b4fb6ae7b40c 
 - Onyx.package.json -> https://github.com/Expensify/react-native-onyx/blob/d7553b95e982ab78f6bb2064f6b0549f0ace94c2/package.json#L16
 - That hash of `expensify-common` uses jQuery 3.3.1 -> https://github.com/Expensify/expensify-common/blob/2e5cff552cf132da90a3fb9756e6b4fb6ae7b40c

Probably someone committed incorrect package-lock state

Reply:
Ah yeah, it looks like we aren't consistently updating repos with expensify-common dependencies after making changes in expensify-common.

I guess it's fine to merge this as-is... cc: @marcaaron just for awareness. I think we need to be more diligent with the follow-up PRs that should be made after making changes to expensify-common.